CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 - Fix `Contacts.presentFormAsync` pre-filling. ([#7285](https://github.com/expo/expo/pull/7285) by [@abdelilah](https://github.com/abdelilah) & [@lukmccall](https://github.com/lukmccall))
 - Removed unknown CLI options `--android-package` and `--ios-bundle-identifier` from docs. ([#7354](https://github.com/expo/expo/pull/7354) by [@ca057](https://github.com/ca057))
 - Fixed `androidNavigationBar.hidden` configuration not remaining applied after backgrounding & foregrounding the app. ([#7770](https://github.com/expo/expo/pull/7770) by [@cruzach](https://github.com/cruzach))
+- Fixed logEvent in `expo-firebase-analytics` for Android. logEvent's optional properties parameter was causing a NPE on Android when not provided. ([#7897](https://github.com/expo/expo/pull/7897) by [@thorbenprimke](https://github.com/thorbenprimke))
```

Comment:
Please add the change-log entry to `packages/expo-firebase-analytics/CHANGELOG.md`.

We've recently changed this approach and we will now be keeping the changelog closer to the package itself.

> - Remember to add a concise description of any user-facing changes to `CHANGELOG.md` file in the package you've changed or [root's CHANGELOG.md](/CHANGELOG.md) if your changes don't apply to any package. This is especially helpful for breaking changes!

Reply:
@IjzerenHein  - thanks for the review. 

I have moved the change-log to the package. I apologize for not dropping it there in the first place. 

I have rebased it against upstream/master. Hopefully the CI will be green. It passed when I tested it locally in the test suite. 

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F3orieTfp1MeFLiBQR2%2Fgiphy.gif&token=pray" data-gitmeme-token="pray"><img src="https://media3.giphy.com/media/3orieTfp1MeFLiBQR2/giphy.gif" title="Created by gitme.me with /pray"/></a>

Reply:
Awesome, thanks!